capellambse/loader/xmltools.py

```diff
@@ -80,6 +82,13 @@ def __init__(
         self.__objclass__: type[t.Any] | None = None
         self.__doc__ = __doc__
 
+    @property
+    def _qualname(self) -> str:
+        """Generate the qualified name of this descriptor."""
+        if self.__objclass__ is None:
+            return f"(unknown {type(self).__name__} - call __set_name__)"
+        return f"{type(self.__objclass__.__name__)}.{self.__name__}"
```

Comment:
I wonder how we endup in the `self.__objclass__ is None` branch and how this will be used later in the `__set__` method of the concrete AttributeProperties (for e.g. [here](https://github.com/DSD-DBS/py-capellambse/pull/116/commits/5023277a4bd45ba3dfea49fe4c80199bac670bd1#diff-21e58b8124b924b3145105940948686ae482d07a2c49f18960638dd80536ab97R208-R215)). Do we give instructions to the user via stdout in this case?

Reply:
`self.__objclass__ is None` if `__set_name__` was never called - which may happen if these descriptor objects are constructed manually and inserted later, instead of being defined in the class. I don't expect that to happen often, and if it does it is a bug in the code that constructs the object.

I don't think we should output anything to stdout/err. If you search for the "magic method" `__set_name__`, you should relatively quickly find the Python docs which explain what it does and when / how it's called.